### PR TITLE
chore(docs): add readme

### PR DESCRIPTION
**Pull request by rokroskar**
Adds a basic readme based on the text that was already in the docs.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -47,7 +58,7 @@ using mlschema-model-converters_ plugin.
 
     export(model, evaluation_measure=(accuracy_score, acc))
 
-Using explicitly the `export` function the details of the supplied model are going to 
+Using explicitly the `export` function the details of the supplied model are going to
```
Comment (Panaetius) [inline README.rst]: This sentence does not read very well.

**Review by Panaetius — DISMISSED** [review]
Thanks. One sentence (you didn't write) could use some love 🙂

File: README.rst (hunk 1 above)
Comment (rokroskar) [reply README.rst]: thanks for flagging it - I tried to improve it

**Review by Panaetius — APPROVED** [review]
👍

**Pull request merged**